tests/main/proxy-no-core/task.yaml

```diff
@@ -11,11 +11,28 @@ details: |
 systems: [ubuntu-16.04-64, ubuntu-18.04-64]
 
 prepare: |
-    # Make sure there is not any .partial for the core snap to be resumed,
-    # otherwise it is not going to use the proxy to download it. The .partial
-    # file is being automatically restored with the snapd state.
+    # Ensure there is no .partial for the core snap to be resumed, otherwise
+    # it is not going to use the proxy to download it. The .partial file is
+    # being automatically restored with the snapd state.
     rm -f /var/lib/snapd/snaps/core_*.partial
 
+    # Ensure core snap is not available in cache, otherwise it is not 
```

Comment:
This scenario is not exercised here, but was specifically tested locally with `SPREAD_SNAPD_DEB_FROM_REPO=false SPREAD_MODIFY_CORE_SNAP_FOR_REEXEC=0 SPREAD_TRUST_TEST_KEYS=false SPREAD_SNAP_REEXEC=0 SPREAD_CORE_CHANNEL=stable`.